server/circleci.go

```diff
@@ -43,20 +44,26 @@ func (s *Server) triggerCircleCiIfNeeded(pr *model.PullRequest) {
 		return
 	}
 
-	// List the files that was modified or added in the PullRequest
-	prFiles, _, err := s.GithubClient.PullRequests.ListFiles(context.Background(), pr.RepoOwner, pr.RepoName, pr.Number, nil)
-	if err != nil {
-		mlog.Error("Error listing the files from a PR", mlog.String("repo", pr.RepoName), mlog.Int("pr", pr.Number), mlog.String("Fullname", pr.FullName), mlog.Err(err))
-		return
-	}
+	blackList, ok := s.Config.FileBlacklist[pr.RepoName]
+	if ok {
+		// List the files that was modified or added in the PullRequest
+		var prFiles []*github.CommitFile
```

Comment:
Any reason this needs to be declared as a separate variable?

Reply:
There needs to be a separate PR for platform-private/mattermod.

Reply:
Did you mean to make this comment on my parent message or the one you replied to?

Reply:
> If this gets auto-deployed when pushed to master, it will break because it expects the config to be in a different way. How do we plan to handle this? @metanerd

To this one.